deploy/utils/config.js

```diff
@@ -0,0 +1,32 @@
+const process = require("process");
+
+// TODO remove, will come from travis
+process.env["AzureClientId"] = "991b63a4-898f-4b6b-b599-db5db9942979";
```

Comment:
Let's remove these values. Please ensure that these credentials are also rotated.